FirebaseMessaging.podspec

```diff
@@ -55,7 +58,7 @@ device, and it is completely free.
   s.osx.framework = 'SystemConfiguration'
   s.weak_framework = 'UserNotifications'
   s.dependency 'FirebaseCore', '~> 6.8'
-  s.dependency 'FirebaseInstanceID', '~> 4.3'
+  s.dependency 'FirebaseInstallations', '~> 1.0'
```

Comment:
Require at least the current 1.5 since we're unlikely to test with anything older.

Reply:
Done. Good catch.